pkg/controller/registry/registry_controller.go

```diff
@@ -108,7 +107,7 @@ func (r *ReconcileRegistry) Reconcile(request reconcile.Request) (reconcile.Resu
 		registry.Status = registriesv1alpha1.RegistryStatus{Phase: registriesv1alpha1.RegistryStatusPhaseTerminating}
 		result = reconcile.Result{Requeue: true}
```

Comment:
this result will never be returned, `return updateRegistryCR` is called directly after this assignment